### PR TITLE
e2e/network/:Use e2eskipper package

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup
/priority backlog

**What this PR does / why we need it**:

e2e/network/:Use e2eskipper package

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/87047

**Special notes for your reviewer**:

/cc @oomichi

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/network/dns_configmap.go [hunk 1]
```diff
@@ -20,12 +20,12 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/onsi/ginkgo"
```
Comment (oomichi) [inline test/e2e/network/dns_configmap.go]: Please don't change this import order, that is unrelated and ginkgo is not in `k8s.io` block..

Comment (tanjunchen) [reply test/e2e/network/dns_configmap.go]: sorry, i will update it.

**Review by oomichi — COMMENTED** [review]
Thanks for doing this.
/lgtm
/approve
